README.template.md

```diff
@@ -0,0 +1,6 @@
+# Roadrunner Rails
+
+This build monitor should be up and running as soon as you set up travis!
+[![Build Status](https://travis-ci.org/department-of-veterans-affairs/roadrunner-rails.svg?branch=master)](https://travis-ci.org/department-of-veterans-affairs/roadrunner-rails)
+
+Tell people about your project!
```

Comment:
Can come later, but if you cut off the top instructions, this could be a template:

https://github.com/department-of-veterans-affairs/roadrunner-rails/blob/quick-start/README.md